File: tests/test-minecraft-osp/src/main/java/pize/tests/voxelgame/main/level/ChunkManager.java
package pize.tests.voxelgame.main.level;

import pize.tests.voxelgame.main.chunk.LevelChunk;
import pize.tests.voxelgame.main.chunk.storage.ChunkPos;

public abstract class ChunkManager{
    
    public abstract LevelChunk getChunk(ChunkPos chunkPos);
    
    public abstract LevelChunk getChunk(int chunkX, int chunkZ);
    
}

File: tests/test-minecraft-osp/src/main/kotlin/pize/tests/voxelgame/main/level/ChunkManager.kt
package pize.tests.voxelgame.main.level

import pize.tests.voxelgame.main.chunk.LevelChunk
import pize.tests.voxelgame.main.chunk.storage.ChunkPos

abstract class ChunkManager {
    abstract fun getChunk(chunkPos: ChunkPos?): LevelChunk?
    abstract fun getChunk(chunkX: Int, chunkZ: Int): LevelChunk?
}
